org-roam-log.el

```diff
@@ -0,0 +1,53 @@
+;;; org-roam-log.el --- Integrations with Org-log -*- coding: utf-8; lexical-binding: t; -*-
+
+;; Copyright © 2022-2022 Jethro Kuan <jethrokuan95@gmail.com>
+
+;; Author: Jethro Kuan <jethrokuan95@gmail.com>
+;; URL: https://github.com/org-roam/org-roam
+;; Keywords: org-mode, roam, convenience
+;; Version: 2.2.1
+;; Package-Requires: ((emacs "26.1") (dash "2.13") (org "9.4") (emacsql "3.0.0") (emacsql-sqlite "1.0.0") (magit-section "3.0.0"))
+
+;; This file is NOT part of GNU Emacs.
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation; either version 3, or (at your option)
+;; any later version.
+;;
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+;;
+;; You should have received a copy of the GNU General Public License
+;; along with GNU Emacs; see the file COPYING.  If not, write to the
+;; Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+;; Boston, MA 02110-1301, USA.
+
+;;; Commentary:
+;;
+;; This module provides integrations with Org-log.
+;;
+;;; Code:
+(require 'org-roam)
+
+(defcustom org-roam-log-setup-hook nil
+  "Hook run when a log for an Org-roam file is setup."
+  :group 'org-roam
+  :type 'hook)
+
+(defun org-roam-log-p ()
+  "Return t if the log buffer is for an Org-roam file, nil otherwise."
+  (and org-log-note-marker
+       (org-roam-file-p (buffer-file-name (marker-buffer org-log-note-marker)))))
+
+(defun org-roam-log--setup ()
+  "Run hooks in `org-roam-log-setup-hook'."
+  (run-hooks 'org-roam-log-setup-hook))
```

Comment:
@jethrokuan , 
Do you intend to do this here? `org-roam-log-p` does not seem to be used?
```
(when (org-roam-log-p)
  (run-hooks 'org-roam-log-setup-hook))
```

Reply:
yeah...